Core/UI/HomeView.cpp

```diff
@@ -40,82 +40,41 @@ HomeView::HomeView(lv_obj_t* parent, HomeViewModel& viewModel) : View(parent, vi
     lv_chart_set_type(lapTimeBarGraph, LV_CHART_TYPE_BAR);
     lv_obj_set_size(lapTimeBarGraph, 200, 150);
     lv_obj_center(lapTimeBarGraph);
-    lv_chart_set_range(lapTimeBarGraph, LV_CHART_AXIS_PRIMARY_Y, 0, 100);
-    lv_chart_set_range(lapTimeBarGraph, LV_CHART_AXIS_SECONDARY_Y, 0, 400);
-    lv_chart_set_point_count(lapTimeBarGraph, 12);
+    lv_chart_set_range(lapTimeBarGraph, LV_CHART_AXIS_PRIMARY_Y, 0, 1010);
+    lv_chart_set_range(lapTimeBarGraph, LV_CHART_AXIS_SECONDARY_Y, 0, 1010);
+//    lv_chart_set_point_count(lapTimeBarGraph, 12);
 
     /*Add ticks and label to every axis*/
     lv_chart_set_axis_tick(lapTimeBarGraph, LV_CHART_AXIS_PRIMARY_X, 10, 5, 12, 3, true, 40);
     lv_chart_set_axis_tick(lapTimeBarGraph, LV_CHART_AXIS_PRIMARY_Y, 10, 5, 6, 2, true, 50);
     lv_chart_set_axis_tick(lapTimeBarGraph, LV_CHART_AXIS_SECONDARY_Y, 10, 5, 3, 4, true, 50);
 
     /*Zoom in a little in X*/
-    lv_chart_set_zoom_x(lapTimeBarGraph, 800);
+//    lv_chart_set_zoom_x(lapTimeBarGraph, 800);
 
     /*Add two data series*/
     lv_chart_series_t * ser1 = lv_chart_add_series(lapTimeBarGraph, lv_palette_lighten(LV_PALETTE_GREEN, 2), LV_CHART_AXIS_PRIMARY_Y);
 
-    lv_coord_t * ser1_array = lv_chart_get_y_array(lapTimeBarGraph, ser1);
```

Comment:
Changed it so that instead of setting the points manually, we just tell the chart to look at an array of values that we make.
The cast is making things a little strange because ms_t is 32bit so it makes 2 chart bands for each time sample. Will fix that in another commit.